### ISSUE TITLE
IOError: No file matching Bubbles_thought.svg in image directory

### ISSUE THREAD

**Issue by austinzh** [3b1b/manim#34]
Hi
Error occurs when I run in OSx
```bash
python extract_scene.py -p old_projects/eola/chapter0.py
```
```bash
Choose number corresponding to desired scene/arguments.
(Use comma separated list for multiple entries)

Choice(s): 1
Animation 1: WriteTextMobject: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:00<00:00, 64.03it/s]
Animation 2: ShowCreationVMobject, etc.: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 40/40 [00:02<00:00, 14.35it/s]



Traceback (most recent call last):
  File "extract_scene.py", line 178, in main
    handle_scene(SceneClass(**scene_kwargs), **config)
  File "/Users/austin/workspace/manim/scene/scene.py", line 38, in __init__
    self.construct(*self.construct_args)
  File "/Users/austin/workspace/manim/./old_projects/eola/chapter0.py", line 112, in construct
    self.to_thought_bubble()
  File "/Users/austin/workspace/manim/./old_projects/eola/chapter0.py", line 152, in to_thought_bubble
    bubble = randy.get_bubble(width = 10)
  File "/Users/austin/workspace/manim/topics/characters.py", line 168, in get_bubble
    bubble = bubble_class(**kwargs)
  File "/Users/austin/workspace/manim/topics/objects.py", line 427, in __init__
    Bubble.__init__(self, **kwargs)
  File "/Users/austin/workspace/manim/topics/objects.py", line 339, in __init__
    SVGMobject.__init__(self, **kwargs)
  File "/Users/austin/workspace/manim/mobject/svg_mobject.py", line 26, in __init__
    self.ensure_valid_file()
  File "/Users/austin/workspace/manim/mobject/svg_mobject.py", line 42, in ensure_valid_file
    raise IOError("No file matching %s in image directory"%self.file_name)
IOError: No file matching Bubbles_thought.svg in image directory
```

**Comment by caph1993** [comment]
Same here

**Comment by ZipWin** [comment]
I have the same problem in Win10

**Comment by eulertour** [comment]
This is fixed after #355.
